pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -164,24 +169,21 @@ type NodeInfo interface {
 
 // CSINodeInfo interface represents anything that can get CSINode object from node ID.
 type CSINodeInfo interface {
-	GetCSINodeInfo(nodeName string) (*storagev1beta1.CSINode, error)
+	GetCSINodeInfo(nodeID string) (*storagev1beta1.CSINode, error)
```

Comment:
I prefer to change the L167 to nodeName

Reply:
I've thought about that too, but I'm not sure which is better. :man_shrugging: 
/cc @Huang-Wei  WDYT?

Reply:
You could search `GetCSINodeInfo` and `GetNodeInfo` globally, and you'll find we always pass `node.Name` or `nodeName` to these functions.

Reply:
Thanks @draveness 
Done! 
Can you leave a `LGTM` for this PR ?

Reply:
Keep it as `nodeName` makes sense to me.

Reply:
Done @Huang-Wei